packages/mobile/src/config/navigation-theme.config.ts

```diff
@@ -48,9 +49,10 @@ const NavigationDarkTheme = {
     icon: ColorPalette.WHITE_100,
     titlePrimary: ColorPalette.WHITE_100,
     titleSecondary: ColorPalette.WHITE_100,
-    yellow: ColorPalette.WHITE_100,
-    blue: ColorPalette.WHITE_100,
+    yellow: ColorPalette.YELLOW_200,
+    blue: ColorPalette.BLUE_100,
     subtitle: ColorPalette.YELLOW_100,
+    logoWhite: ColorPalette.WHITE_100,
```

Comment:
Remove all of these

Reply:
delete logoWhite, but leave blue and yellow, as it maybe in use somewhere